core-bundle/src/Command/ResizeImagesCommand.php

```diff
@@ -85,181 +102,250 @@ protected function configure(): void
     {
         $this
             ->addOption('time-limit', 'l', InputOption::VALUE_OPTIONAL, 'Time limit in seconds', '0')
-            ->addOption('concurrent', 'c', InputOption::VALUE_OPTIONAL, 'Run multiple processes concurrently', '1')
-            ->addOption('throttle', 't', InputOption::VALUE_OPTIONAL, 'Pause between resizes to limit CPU utilization, 0.1 relates to 10% CPU usage', '1')
+            ->addOption('concurrent', 'c', InputOption::VALUE_OPTIONAL, 'Run multiple processes concurrently with a value larger than 1 or pause between resizes to limit CPU utilization with values lower than 1.0', '1')
+            ->addOption('throttle', 't', InputOption::VALUE_OPTIONAL, '(Deprecated) Use the concurrent option instead', '1')
+            ->addOption('image', null, InputArgument::OPTIONAL, 'Image name to resize a single image')
             ->setDescription('Resizes deferred images that have not been processed yet.')
         ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $this->io = new SymfonyStyle($input, $output->section());
+
         if (!$this->resizer instanceof DeferredResizerInterface) {
             throw new \RuntimeException('Deferred resizer not available');
         }
 
-        $timeLimit = (float) $input->getOption('time-limit');
-        $throttle = (float) $input->getOption('throttle');
-        $concurrent = (int) $input->getOption('concurrent');
-
-        if ($timeLimit < 0) {
-            throw new InvalidArgumentException(sprintf('Time-limit value "%s" is invalid.', $timeLimit));
-        }
-
-        if ($throttle < 0.001 || $throttle > 1) {
-            throw new InvalidArgumentException(sprintf('Throttle value "%s" is invalid.', $throttle));
-        }
-
-        if ($concurrent < 1) {
-            throw new InvalidArgumentException(sprintf('Concurrent value "%s" is invalid.', $concurrent));
+        if (!$output instanceof ConsoleOutputInterface) {
```

Comment:
Why is this needed? Imo none of the `ConsoleOutputInterface`'s methods are in use.

Reply:
Fixed in 54c1e902208d58505e1eac0eeb4b8d31f718b1ff